macros/models/_sector/addresses_events/addresses_events_first_funded_by.sql

```diff
@@ -0,0 +1,23 @@
+{% macro addresses_events_first_funded_by(blockchain, traces) %}
```

Comment:
```suggestion
{% macro addresses_events_first_funded_by(blockchain) %}
```
The traces variable wasn't used.